src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const shortTerm = 3;
+  const shortTermDiscount = 20;
+  const longTerm = 7;
+  const longTermDiscount = 50;
+  const price = 40;
+
+  if (days < shortTerm) {
+    return days * price;
+  } else if (days < longTerm) {
+    return days * price - shortTermDiscount;
```

Comment:
The condition `else if (days < longTerm)` applies a fixed short-term discount for rentals between 3 and 6 days. Ensure this logic aligns with the intended discount strategy. If the discount should be applied per day beyond a certain threshold, the calculation needs adjustment.